apps/meteor/client/sidebar/header/hooks/useEncryptedRoomDescription.tsx

```diff
@@ -4,19 +4,19 @@ import { useTranslation } from 'react-i18next';
 export const useEncryptedRoomDescription = (roomType: 'channel' | 'team') => {
```

Comment:
I saw that `createTeamModal` implements this hook as well, should that modal be affected by these changes too?

Reply:
We can do that. But I think, wont be necessary since its what I am working on right now. So the hook and the tests will be updated as part of it.